lib/zendesk_apps_support/package.rb

```diff
@@ -190,7 +190,7 @@ def has_location?
     deprecate :has_location?, 'manifest.location?', 2016, 9
 
     def has_file?(path)
-      File.exist?(path_to(path))
+      File.file?(path_to(path))
```

Comment:
it's also used for validating location url's:
```
  location: { support: { top_bar: "my_page.html" } }
```
can that be a path to a folder?